W/Winston/Compat.toml

```diff
@@ -28,17 +28,25 @@ Tk = "0.5.2"
 julia = "0.5-0.7"
 
 ["0.13-0.14"]
+StatsBase = "0.13.1-0.29"
+
+["0.13-0.15"]
 Graphics = "0.2-0.4"
 IniFile = "0.3-0.5"
 NaNMath = "0.2.4-0.3"
-StatsBase = "0.13.1-0.29"
 
 ["0.14"]
-Compat = "0.60-2.1"
 Gtk = "0.1-0.16"
 Reexport = "0.0-0.2"
 julia = "0.7-1.3"
 
+["0.14-0.15"]
+Compat = "0.60-2.1"
+
+["0.15"]
+StatsBase = "0.13.1-0.30"
+julia = "0.7-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.